test/sanity/README.md

```diff
@@ -1,52 +1,45 @@
-# Kubernetes Sanity Script Test
+# Sanity Tests for CSI PowerFlex
 
 This test runs the Kubernetes sanity test at https://github.com/kubernetes-csi/csi-test.
-The version was v3.1.0
 
-To run the test:
-
-	1. run `git clone https://github.com/kubernetes-csi/csi-test.git`
-	2. Check out tag v3.1.0
-	3. Cd to csi-test/cmd/csi-sanity
-	4. run `make clean install`
-	5. Make sure env.sh(in csi-vxflexos) is up to date
-	6. Cd to csi-vxflexos/test/sanity and edit secrets.yaml and volParams.yaml to have the correct storage pool parameter 
-	7. Build csi-vxflexos ("make clean build" in csi-vxflexos)
-	8. Run start_driver.sh 
-	9. Open up another window and bring it to csi-vxflexos/test/sanity
-	10. Use run.sh to run the tests once
-		Use debug_help.sh to run an sh file x amount of times
-			debug_help.sh run.sh 100 30 runs run.sh 100 times, with thirty second breaks inbetween each run
-			This is useful for catching non-consitent errors. A 30 second wait time is a good idea when running 
-			the tests over 5 times with the script, to allow enough time for the tests to cleanup. 
-
-
-##Excluded Tests
-
-1.pagination should detect volumes added between pages and accept tokens when the last volume from a page is deleted
```

Comment:
nitpicking here, Start first letter with upper case. Pagination here. likewise in a few other places.

Reply:
I've left the `excluded tests` points lower case to match the regex that is used in the scripts. I've also wrapped it in a code format to make it a bit more explicit (see below).